Makefile

```diff
@@ -67,10 +67,71 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
-.PHONY: runlocal-rp
-runlocal-rp:
-	go run -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro rp
+# Target to create Podman secrets
+.PHONY: podman-secrets
+podman-secrets: aks.kubeconfig
+	podman secret rm --ignore aks.kubeconfig
```

Comment:
Can't this be changed to `docker` everywhere to remain consistent through out the Makefile?